classes/Project/ProjectFactory.js

```diff
@@ -181,6 +184,284 @@ export default class ProjectFactory {
+      })
+    }
+  }
+
+  static async copyProject(projectId, creator) {
+    if (!projectId) {
+      throw {
+        status: 400,
+        message: "No project ID provided"
+      }
+    }
+
+    const project = await ProjectFactory.loadAsUser(projectId, creator)
+    if (!project) {
+      throw {
+        status: 404,
+        message: "Project not found"
+      }
+    }
+
+    let copiedProject = this.copiedProjectConfig(project, database, creator)
+    await this.cloneLayers(project, copiedProject, database, true)
+    copiedProject._lastModified = copiedProject.layers[0]?.pages[0]?.id.split('/').pop()
+    const copiedGroup = await this.cloneGroup(project, creator, { 'Group Members': true })
+    await this.cloneHotkeys(project._id, copiedProject._id)
+    return await new Project().create({ ...copiedProject, creator, group: copiedGroup._id })
+  }
+
+  static async cloneWithoutAnnotations(projectId, creator) {
+    if (!projectId) {
+      throw {
+        status: 400,
+        message: "No project ID provided"
+      }
+    }
+
+    const project = await ProjectFactory.loadAsUser(projectId, creator)
+    if (!project) {
+      throw {
+        status: 404,
+        message: "Project not found"
+      }
+    }
+
+    let copiedProject = this.copiedProjectConfig(project, database, creator)
+    await this.cloneLayers(project, copiedProject, database, false)
+    copiedProject._lastModified = copiedProject.layers[0]?.pages[0]?.id.split('/').pop()
+    const copiedGroup = await this.cloneGroup(project, creator, { 'Group Members': true })
+    await this.cloneHotkeys(project._id, copiedProject._id)
+    return await new Project().create({ ...copiedProject, creator, group: copiedGroup._id })
+  }
+
+  static async cloneWithGroup(projectId, creator) {
+    if (!projectId) {
+      throw {
+        status: 400,
+        message: "No project ID provided"
+      }
+    }
+    const project = await ProjectFactory.loadAsUser(projectId, creator)
+    if (!project) {
+      throw {
+        status: 404,
+        message: "Project not found"
+      }
+    }
+
+    let copiedProject = this.copiedProjectConfig(project, database, creator, { 'Metadata': false, 'Tools': false })
+    await this.cloneLayers(project, copiedProject, database, true)
+    copiedProject._lastModified = copiedProject.layers[0]?.pages[0]?.id.split('/').pop()
+    const copiedGroup = await this.cloneGroup(project, creator, { 'Group Members': true })
+    return await new Project().create({ ...copiedProject, creator, group: copiedGroup._id })
+  }
+
+  static async cloneWithCustomizations(projectId, creator, modules) {
+    if (!projectId) {
+      throw {
+        status: 400,
+        message: "No project ID provided"
+      }
+    }
+
+    if (!modules || typeof modules !== 'object') {
+      throw {
+        status: 400,
+        message: "Modules must be an object"
+      }
+    }
+
+    const project = await ProjectFactory.loadAsUser(projectId, creator)
```

Comment:
Definitely don't loadAsUser() here. This should be a copy of the database object, not the projection for the User.